apps/estuary/test/integration/application_start_test.exs

```diff
@@ -1,11 +1,68 @@
 defmodule Estuary.StartTest do
   use ExUnit.Case
+  use Placebo
   use Divo
+  import ExUnit.CaptureLog
+  require Logger
 
   @elsa_endpoint Application.get_env(:estuary, :elsa_endpoint)
   @event_stream_topic Application.get_env(:estuary, :event_stream_topic)
 
-  test "Topic is created when Estuary starts" do
-    assert Elsa.Topic.exists?(@elsa_endpoint, @event_stream_topic)
+  describe "Estuary.MessageHandler receives messages sent to the event stream" do
```

Comment:
Shall we consider moving the tests to separate file if those are not related to startup of the application

Reply:
Yeah, it'd be good to either have all the tests in the same file with a general name (for testing speed) or to have them broken out by feature/ area under test. (Slower but clearer).

Reply:
Done

Reply:
done